pkg/engine/systemd.go

```diff
@@ -33,14 +33,14 @@ func systemdPodman(path string) error {
 	}
 	s.Command = []string{"sh", "-c", "cp " + copyFile}
 	s.Mounts = []specs.Mount{{Source: systemdLocation, Destination: systemdLocation, Type: "bind", Options: []string{"rw"}}}
-	s.Volumes = []*specgen.NamedVolume{{Name: "harpoon-volume", Dest: "/opt", Options: []string{"ro"}}}
+	s.Volumes = []*specgen.NamedVolume{{Name: harpoonVolume, Dest: "/opt", Options: []string{"ro"}}}
```

Comment:
@sallyom I'm wondering if we should patch the test cases and readme to change the name to harpoonVolume from harpoon-volume.

Reply:
for now, we can keep it harpoon-volume, and open an issue for the **bug** where **volume-name passed in has to be "harpoon-volume",** so have to make the program smarter. this is separate from this PR, though.
In resolving this bug, harpoonVolume  will default to "harpoon-volume" but will pick up any override from CLI or config. 
What we need is a CLI (that may only read from the config file, or may have command line overrides)

Reply:
I see the constant now. My bad! Still learning